rebar.config

```diff
@@ -1,7 +1,7 @@
 {deps, [
     {lager, {git, "https://github.com/basho/lager", {tag, "3.2.4"}}},
     {jsx, {git, "https://github.com/talentdeficit/jsx.git", {tag, "master"}}},
-    {erldns, {git, "https://github.com/dcos/erl-dns.git", "deprecate-records_by_type"}},
+    {erldns, {git, "https://github.com/dcos/erl-dns.git", master}},
```

Comment:
yeah, that is a recommendation from rebar.config. We try to change is as and when we update a particular reference. So, yeah it is not complete